### PR TITLE
Ensure tmp directory gets created

### PR DESCRIPTION
**Pull request by markelliot**

### CODE REVIEW

File: src/main/groovy/com/palantir/gradle/javadist/tasks/DistTarTask.groovy [hunk 1]
```diff
@@ -43,7 +43,10 @@ class DistTarTask extends Tar {
                 }
             }
 
-            new File(project.projectDir, "var/data/tmp").mkdirs()
+            new File(project.buildDir, "distributions/var/data/tmp").mkdirs()
+            from ("${project.buildDir}/distributions/var/data") {
```
Comment (uschi2000) [inline src/main/groovy/com/palantir/gradle/javadist/tasks/DistTarTask.groovy]: should add to readme that `excludeFromVar 'data'` has no effect.

Comment (markelliot) [reply src/main/groovy/com/palantir/gradle/javadist/tasks/DistTarTask.groovy]: Can clarify in the readme, but the exclusion is copying `${projectDir}/var/data`, which this still respects.

**Comment by uschi2000** [comment]
the coveralls integration seems borked?

**Comment by uschi2000** [comment]
(not only on this pr)

**Comment by markelliot** [comment]
yeah, we reset our 3rd party integrations, going to see if i can fix

**Comment by markelliot** [comment]
Yeah, status API requests don't seem to be going through. Will hit up our admin team to see what's up.

**Pull request merged**